locales/nl-NL.json

```diff
@@ -163,13 +169,13 @@
     "zen_mode": "Zen Modus"
   },
   "notification": {
-    "favourited_post": "vindt jouw post favoriet",
+    "favourited_post": "heeft jouw post als favoriet opgeslagen",
```

Comment:
Thanks for the PR! It would be good if there is a way to shorten this one, I think it may look strange in the notifications UI

Reply:
Hard to do in Dutch. An alternative would be "vindt jouw post leuk", but that foregoes any occurrence of the word "favourite".

Reply:
I'm not 100% sure what the context looks like for this specific line.
It might work without the first part `heeft jouw post`?

*Could someone send a screenshot of this line in the GUI?*

Reply:
![Screenshot 2023-05-03 at 11 19 37](https://user-images.githubusercontent.com/851445/235878247-4a5d3e64-fbb0-468b-828d-17f79d474d24.png)
@mxcop

Reply:
Hmmm, "heeft als favoriet opgeslagen" I think that might also work.
However I do think "vindt jouw post leuk" would make the most sense, even if it foregoes the use of "favourite".

Reply:
Yeah, agree, reads nicer. I'll adjust.